### DISCUSSION TITLE
Expired: -käännös varauksissa

### DISCUSSION THREAD

**Discussion by hanhisalok** [KohaSuomi/Koha-translations#25] [Yleinen]
Tieteelliset kirjastot keskusteli tänään kokouksessa varauksen näkymässä olevasta käännöksestä "Expired:" (/holds_table.inc:135). Tulee esiin taulukossa, kun on varauksen viimeinen noutopäivä. Nyt se käännetty suodaan "Vanhentunut:", mutta mietimme, että selkeämpi voisi olla "Viimeinen noutopäivä:"

Vaihdan sen, jos kellään ei ole sitä mitään vastaan.

Ellei jollain ole tietoa, että tuo ilmenee myös jossain muussa kohtaa.

**Comment by AnneliO** [comment]
Oisko sulla Katariina kuvakaappausta tuosta näkymästä, kun en nyt hoksaa, miten se näkyy enkä saa sandboxissa tehtyä varausta, joka vanhenee tänään? :)

**Reply by AnneliO** [reply]
Ja jostain syystä tuota ei meidän Kohista löydy. Se on varmaan tullut jonkin vähän uudemman päivityksen/korjauksen mukana kuin mitä meillä on asennettuna.

**Reply by hanhisalok** [reply]
![Capture](https://github.com/KohaSuomi/Koha-translations/assets/49274024/88510baa-9098-4dbd-9ed0-841d582e0839)

Voi olla, että se on sitten vaan meillä näkyvissä. Tulee tosiaan näkyviin sinä päivänä, kun on varauksen vanhenemispäivä. Tästä tosin oli meillä yhteisesti vielä puhetta, että saattaa olla myös koodissa korjaamista. Koska nythän viimeisenä noutopäivänä ei edes voi muuttaa tuota päivämäärää (ellei peruuta koko odottaa-tilaa).

**Reply by AnneliO** [reply]
Hmm.. joo, meillä ei ole tuollaista näkymää, mutta toisaalta, meillä on lisätty tuolle sivulle sivutus, joka muuttaa näkymää muutenkin. Eli sekin voi selittää, miksi meillä ei ole tuollaista. Meidän näkymä on tällainen:
![kuva](https://github.com/KohaSuomi/Koha-translations/assets/33121325/e6674372-7765-4fef-afe9-59be72b4b906)

Minusta tuossa kyllä tosiaan on jonkinlainen looginen virhe, koska varaushan on voimassa vielä viimeisenä noutopäivänä. Ei se ole vielä silloin "expired", kuten tuo nyt väittää. Minun logiikka sanoo, että tuollainen pitäisi näkyä vasta viimeistä noutopäivää seuraavana päivänä ja vain, jos käytössä ei ole cancel_expired_holds.pl -cron job.

Vai onko tuohon tullut jokin muutos, että expirationdate on oikeasti se päivä, jolloin varaus vanhenee? Eikä ole viimeinen noutopäivä? Kohan schema kyllä sanoo, että expiratiodate = *the date the hold expires (calculated value)* mikä kyllä minusta tarkoittaisi päivää, jolloin varaus vanhenee ja sitä edeltävä päivä olisi sitten viimeinen noutopäivä. Otapa näistä selvää. :D

**Reply by hanhisalok** [reply]
Käsittääkseni jotkut kirjastot tätä oli testanneetkin ja ilmeisesti asiakkaan päässä tuo näkyy oikein ja on oikeasti edelleen noudettavissa. Mutta ehkäpä en ainakaan vielä käännä tätä, kun alkaa vaikuttaa tosiaan virheeltä. Voipi olla, että kehittäjät tuon korjaakin (piakkoin), kun tieto on sinne jo asiasta mennyt.
